src/compas/__init__.py

```diff
@@ -136,6 +136,8 @@
 
 __all_plugins__ = [
     "compas.geometry.booleans.booleans_shapely",
+    "compas.geometry.curves.nurbs_",
+    "compas.geometry.surfaces.nurbs_",
```

Comment:
why the `_` as suffix instead of prefix?

Reply:
like `nurbs_numpy` but then for `nurbs_python`. figured it could just be `nurbs_` instead...

Reply:
ah, i see :)